amethyst_network/src/lib.rs

```diff
@@ -50,9 +56,9 @@ where
 }
 
 /// Attempts to deserialize an event from the raw byte data.
-fn deserialize_event<T>(data: &[u8]) -> Result<NetEvent<T>, Box<ErrorKind>>
+fn deserialize_event<T>(data: &[u8]) -> Result<NetEvent<T>>
```

Comment:
Non-public function so can use `//` not `///`

Reply:
Should probably still prefer `///` since they are contextual to the function and used by e.g. RLS. Don't know if that works as intended with `//`.